modules/runner/src/integrationTest/java/org/apache/ignite/internal/rest/ItNotInitializedClusterRestTest.java

```diff
@@ -188,4 +189,16 @@ void clusterState() throws IOException, InterruptedException {
                         is("Cluster not initialized. Call /management/v1/cluster/init in order to initialize cluster"))
         );
     }
+
+    @Test
+    @DisplayName("Node version is available on not initialized cluster")
+    void nodeVersion() throws IOException, InterruptedException {
+        // When GET /management/v1/node/version/
+        HttpResponse<String> response = client.send(get("/management/v1/node/version/"), BodyHandlers.ofString());
+
+        // Then
+        assertThat(response.statusCode(), is(200));
+        // And version is a semver
+        assertThat(response.body(), matchesRegex("[1-9]\\d*\\.\\d+\\.\\d+(?:-[a-zA-Z0-9]+)?"));
```

Comment:
Perhaps, it makes sense to use `IGNITE_SEMVER_REGEX` here as well. What do you think?